src/server/app/api/controllers/authController.js

```diff
@@ -20,6 +20,22 @@ const authController = {
       return Boom.badRequest(err.message || "Error creating user login token.");
     }
   },
+  async validateAuthToken(req, h) {
+    let validation;
+    try {
+      if (!req.payload) throw new Error("Missing authentication payload.");
+      if (!req.payload.token) throw new Error("Missing authentication token.");
+      if (!req.payload.username) throw new Error("Auth payload missing username.");
+      validation = await validateToken(req.payload.username, req.payload.token);
+      if (Bounce.isError(validation)) throw new Error(validation.message || "No validation returned.");
+      if (!validation.isValid) throw new Error("Auth token not valid.");
+      const response = h.response(validation);
+      return response;
```

Comment:
Same points here about whitespace seperating the `if` statements and the actual logic of the function. What I'm basically trying to suggest is a clear visual separation between the actual logic of the function and all of the guard statements.

Also, could do `return h.response(validation);`